tests/api/worlds/altimeter.wbt

```diff
@@ -0,0 +1,58 @@
+#VRML_SIM R2021a utf8
+WorldInfo {
+  coordinateSystem "NUE"

```

Comment:
```suggestion
```
We want to push ENU as a default coordinate system. Therefore, it would be nice that new tests use ENU (it is more important that a device works properly for ENU than for NUE)